ext/standard/tests/http/gh16810.phpt

```diff
@@ -6,11 +6,18 @@ allow_url_fopen=1
 <?php if (PHP_INT_SIZE != 8) die("skip this test is for 64bit platform only"); ?>
 --FILE--
 <?php
-$uri = "http://www.example.com";
```

Comment:
Could you please fix the `--TEST--` section above? That's apparently a copy&paste issue; should be:
````
GH-16809 (fopen HTTP wrapper timeout stream context option overflow)
````
Test name should be gh16809.phpt accordingly, but it might be better to leave that as is.

Reply:
ah yeah good spot. Done.